### PR TITLE
Port tests over to GitHub Actions

### PR DESCRIPTION
**Pull request by neko314**
This PR is dummy to check if GitHub Actions work and jobs succeed for https://github.com/codetriage/CodeTriage/pull/1724

### CODE REVIEW

**Comment by neko314** [comment]
@gregmolnar

Checking passes with Ruby 3.0 and 3.1 at this commit https://github.com/gregmolnar/CodeTriage/pull/1/commits/ef8352767b1a6c602ecba01906f24ddc41096603 .
I think all commits are better to merge to one commit finally but I make new change new commits right now so that you know what I think, try or want to do. 

Test yet fails with Ruby 2.7.
I'll see this problem later.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Main
+on: [push, pull_request]
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7, "3.0", 3.1]
```
Comment (gregmolnar) [inline .github/workflows/main.yml]: @neko314 Actually we need ruby 2.7.2 here:
`ruby: [2.7.2, 3.0, 3.1]`

Comment (neko314) [reply .github/workflows/main.yml]: Thank you. I fixed it at https://github.com/gregmolnar/CodeTriage/pull/1/commits/ff2162bb23e3db17881bace07b8b9026b37ab88b

**Comment by neko314** [comment]
https://github.com/codetriage/CodeTriage/pull/1724 are merged!🎉
Then, I close this one.
Thank you so much for your pair working. It was very exciting andI had a lot of fun🙂

**Pull request closed without merging**